content/en/platform/corda/5.2/deploying-operating/deployment/deploying/_index.md

```diff
@@ -284,15 +285,11 @@ As with the number of replicas, you may need to adjust these values based on tes
 
 #### Recommended Infrastructure
 
-For an AWS topology, we recommend the following initial configuration:
+For an AWS topology, R3 recommends the following initial configuration:
 
-* Kubernetes: For a cluster with a single replica of each worker, a Kubernetes cluster with two `t3.2xlarge` nodes is
-  a reasonable starting point. For a cluster with three replicas of each worker, extend that to four nodes.
-* RDS PostgreSQL: `db.r5.large` instance size is sufficient for both a Corda cluster with a single replica of each worker
-  and three replicas of each worker, subject to the persistence requirements of any {{< tooltip >}}CorDapp{{< /tooltip >}} running in the cluster.
-* MSK: For a cluster with a single replica of each worker and a topic replica count of three, a Kafka cluster of three
-  `kafka.t3.small` instances may suffice. In a high-availability topology with three replicas of each worker and a topic replica count
-  of three, we recommend five brokers using at least `kafka.m5.large` instances.
+* Kubernetes: For a cluster with a single replica of each worker, a Kubernetes cluster with two `t3.2xlarge` nodes is a reasonable starting point. For a cluster with three replicas of each worker, extend that to four nodes.
+* RDS PostgreSQL: `db.r5.large` instance size is sufficient for both a Corda cluster with a single replica of each worker and three replicas of each worker, subject to the persistence requirements of any {{< tooltip >}}CorDapp{{< /tooltip >}} running in the cluster.
+* MSK: For a cluster with a single replica of each worker and a topic replica count of three, a Kafka cluster of three `kafka.t3.small` instances may suffice. In a high-availability topology with three replicas of each worker and a topic replica count of three, R3 recommends five brokers using at least `kafka.m5.large` instances.
```

Comment:
@jujoramos - this information dates back to 5.0. Does the ORR team have any more up-to-date recommendations here? Or would it be better to remove this section (and make this information available internally)?

Reply:
Removed ✅ .